src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -29,31 +29,40 @@ export const CentralImportDuplicateSearch = ({ duplicateSearchFilters }: Central
   duplicateSearchFilters.yearPublished.length > 0 &&
     queryArray.push(`${DescriptionFieldNames.PublicationYear}:"${duplicateSearchFilters.yearPublished}"`);
 
-  const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')})` : '';
+  const searchQuery = queryArray.length > 0 ? `query=(${queryArray.join(' AND ')}&results=${maxHits})` : '';
 
-  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
-    url: `${SearchApiPath.Registrations}?${searchQuery}&results=${maxHits}`,
-    errorMessage: t('feedback.error.search'),
+  const importCandidateQuery = useQuery({
+    queryKey: ['importCandidates', searchQuery],
+    queryFn: () => fetchImportCandidates(),
+    onError: () =>
+      dispatch(
+        setNotification({
+          message: t('feedback.error.get_messages'),
+          variant: 'error',
+        })
+      ),
```

Comment:
`onError` callback ble nylig market deprecated, så vi prøver å legge ny kode til å bruke ny flyt for feilmeldinger. Si fra om du vil ha mer forklaring på hvordan akkurat det virker 

```suggestion
    meta: { errorMessage: t('feedback.error.get_registrations') },
```

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/2a3bcbf3-d678-4ee5-b8d9-19e2290080fb)